dashboard/app/models/programming_environment.rb

```diff
@@ -8,6 +8,37 @@
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
+#
+# @attr [String] editor_type - Type of editor one of the following: 'text-based', 'droplet', 'blockly'
```

Comment:
Is "text-based" just for weblab? If so, what do you think about changing it to "bramble" (the weblab editor)?

Reply:
It will be used for CSA too in the future.

Reply:
My understanding is that we wanted to use editor_type to determine how to show the blocks/text in markdown. So droplet blocks would look different than blockly blocks etc.